nucleus/pom.xml

```diff
@@ -48,6 +48,11 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-hateoas</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-configuration-processor</artifactId>
```

Comment:
Just for future, you don't actually need this dependency for what we're doing.
This processor dependency only becomes handy when you have a Spring auto configuration project and you want to enable auto completion for the configuration properties in IDE's that can use this.

See [here](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#configuration-metadata-providing-manual-hints)

Can keep it, just FYI.